dxr/testing.py

```diff
@@ -267,11 +265,12 @@ def config_input(cls, config_dir_path):
 
     @classmethod
     def teardown_class(cls):
-        if cls.should_delete_instance:
+        if cls.stop_for_interaction:
+            print "Pausing for interaction at 0.0.0.0:8000..."
```

Comment:
Flask already catches ctrl-c and returns cleanly from make_app.run(), so I removed the else clause to allow fall-through to the cleanup code.